website/docs/docs/stubs.md

```diff
@@ -21,10 +21,8 @@ Hosted API stubs are useful for several use cases, such as:
 | 1       | ✅         |
 | 1.1     | ✅         |
 | 2       | ✅         |
-| 3       | ❌         |
-| 4+      | ❌         |
-
-_NOTE: Whilst currently only versions 1 and 2 of the [pact specification](https://github.com/pact-foundation/pact-specification/) are currently fully supported. Pacts using the v3 format or beyond may be used, however any matching rules will be ignored._
+| 3       | ✅         |
+| 4+      | ✅         |
```

Comment:
Should note that plugins are not supported

Reply:
Thanks for the reminder. I'll add to the limitations in the section at the bottom of the page.